app.py

```diff
@@ -234,25 +254,29 @@ def parse_date(d):
 
 
 # Takes list of events and returns list of events occuring in specified date range
-def filter_events_by_date(events, start_date_str=datetime.datetime.now(datetime.timezone.utc), end_date_str=None):
+    def filter_events_by_date(events, start_date_str=datetime.datetime.now(datetime.timezone.utc), end_date_str=None):
+        
+        # number of days specified in config
+        days = config.get('past_events', 'past_days')
```

Comment:
Probably good to name this the same as in the other method, days_in_the_past probably reads well, or something like it.